internal/node/component/info/component.go

```diff
@@ -49,6 +49,7 @@ func NewComponent(_ context.Context, apiServer *echo.Echo, config *config.File,
 	apiServer.GET("/version", c.GetVersion)
 	apiServer.GET("/activity_count", c.GetActivityCount)
 	apiServer.GET("/workers_status", c.GetWorkersStatus)
+	apiServer.GET("/workers_info", c.GetWorkersInfo)
```

Comment:
the documentation is missing a description for workers_info API

Reply:
openapi doc added.